vendor/views/config.py

```diff
@@ -0,0 +1,280 @@
+        context['form'].fields['site'].disabled = True
+        context['cancel_url'] = reverse('vendor_admin:manager-config-processor-list')
+
+        return context
+
+
+    def post(self, request, *args, **kwargs):
+        # Get the SiteConfigModel instance passed through the url
+        self.object = self.get_object()
+        context = super().get_context_data(**kwargs)
+
+        form = self.get_form_class()(request.POST)
+        # Make this site field required False since you can only update the account_number
+        form.fields['site'].required = False
+
+        if not form.is_valid():
+            context['form'] = form
+            return render(request, self.template_name, context)
+        
+        processor_config = PaymentProcessorSiteConfig(self.object.site)
+        processor_config.save(form.cleaned_data['payment_processor'], "payment_processor")
+
+        return redirect("vendor_admin:manager-config-processor-list")
+
+
+class StripeConnectAccountCreateConfigView(FormView):
+    template_name = 'vendor/manage/config.html'
+    form_class = StripeConnectAccountForm
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        stripe_config = StripeConnectAccountConfig()
+        form = self.get_form()
+
+        existing_stripe_configs = SiteConfigModel.objects.filter(key=stripe_config.key)
+        sites = Site.objects.exclude(pk__in=[config.site.pk for config in existing_stripe_configs])
+        form.fields['site'].queryset = sites
+
+        context['title'] = _("Stripe Connect Account")
+        context['form'] = form
+        context['cancel_url'] = reverse('vendor_admin:manager-config-stripe-connect-list')
+
+        return context
+
+    def form_valid(self, form):
+        site = form.cleaned_data['site']
+        
+        stripe_connect = StripeConnectAccountConfig(site)
+        stripe_connect.save(form.cleaned_data['account_number'], "stripe_connect_account")
+
+        return redirect("vendor_admin:manager-config-stripe-connect-list")
+
+
+class StripeConnectAccountConfigListView(ListView):
+    template_name = 'vendor/manage/config_list.html'
+    model = SiteConfigModel
+
+    def get_queryset(self):
+        stripe_config = StripeConnectAccountConfig()
+        
+        return self.model.objects.filter(key=stripe_config.key)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        stripe_config = StripeConnectAccountConfig()
+
+        context['title'] = 'Stripe Connect Accounts'
+        context['config_key'] = stripe_config.key
+        context['new_url'] = reverse('vendor_admin:manager-config-stripe-connect-create')
+
+        return context
+
+
+class StripeConnectAccountUpdateConfigView(UpdateView):
+    template_name = 'vendor/manage/config.html'
+    model = SiteConfigModel
+    form_class = StripeConnectAccountForm
+
+    def get_form(self, **kwargs):
```

Comment:
don't think you need to declare get_form

Reply:
removed kwargs

Reply:
I see what you mean duh. thanks

Reply:
I remember now. I am over righting it because I want it to ignore built in functionality it has whit the UpdateView Class